beard/clustering/blocking.py

```diff
@@ -50,10 +50,6 @@ def _parallel_fit(fit_, partial_fit_, estimator, verbose, data_queue,
     # by the main process
     status, block, existing_clusterer = data_queue.get()
 
-    if status == 'end':
-        # Special case where there are more processes than blocks
-        result_queue.put((None, None))
-
```

Comment:
These lines were added for a situation where number of processes is bigger than number of blocks. They are obsolete now after I merged 97e930f9113dd416145e9e559b7e863a73eafc86